astropy/stats/bls/core.py

```diff
@@ -745,26 +744,3 @@ def __repr__(self):
 
     def __dir__(self):
         return list(self.keys())
-
-    def assert_allclose(self, other, **kwargs):
```

Comment:
I still think something like `allclose` that returns boolean might be nice for users, but I guess it can be implemented (again) if there is demand. Thanks!